README.md

```diff
@@ -19,34 +19,42 @@ Using the Spatial Audio capability, you now have the possibility to create priva
 ```ts
 const zone = {
     origin: {
-        x: 0, y: 0, z: 0
+        x: 0,
+        y: 0,
+        z: 0,
     },
     dimension: {
-        x: 100, y: 100, z: 100
+        x: 100,
+        y: 100,
+        z: 100,
     },
     scale: {
-        x: 10, y: 10, z: 10,
+        x: 10,
+        y: 10,
+        z: 10,
     },
 };
 // Create a new private zone with the rules set above in the zone object
-const zoneId = await VoxeetSDKExt.spatialAudio.createPrivateZone(zone);
+const zoneId = await VoxeetSDKExt.privateZones.createPrivateZone(zone);
```

Comment:
naming: is it a bit redundant: privateZones.createPrivateZone ? 
Could we have something like this: privateZones.createZones.

Reply:
I removed the word `Private` in the function names.